modules/datahandling/projectrequests.go

```diff
@@ -417,6 +413,23 @@ func (p projectLookupRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 	return []dhClosure{toSenderClosure{msg: res}}, nil
 }
 
+func projectLookup(senderID string, projectID int64, db dbfs.DBFS) (projectLookupResult, error) {
+	var resultData projectLookupResult
+
+	name, permissions, err := db.MySQLProjectLookup(projectID, senderID)
+
+	if err != nil {
+		return resultData, err
+	}
+
+	resultData = projectLookupResult{
+		ProjectID:   projectID,
+		Name:        name,
+		Permissions: permissions}
```

Comment:
this should look like 
```
resultData = projectLookupResult{
 		ProjectID:   projectID,
 		Name:        name,
 		Permissions: permissions,
}
```
I know it's kinda nitpicky but it's what we've been going with.